### ISSUE TITLE
[Bug]: Panic when `concatenateModules` and `CssExtractRspackPlugin` used

### ISSUE THREAD

**Issue by JohnDaly** [web-infra-dev/rspack#6927]
### System Info

npmPackages:
  @rspack/cli: 0.7.5 => 0.7.5
  @rspack/core: 0.7.5 => 0.7.5

### Details

When attempting to do a production build in the [reproduction repo](https://github.com/JohnDaly/rspack-concatenate-modules-panic-repro) with `concatenateModules: true` and `CssExtractRspackPlugin`, the `rspack` process panics with the following error:
```
Panic occurred at runtime. Please file an issue on GitHub with the backtrace below: https://github.com/web-infra-dev/rspack/issues
Message:  Cannot get final name for export 'default'
Location: crates/rspack_core/src/concatenated_module.rs:2136

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 1: _napi_register_module_v1
    at <unknown source file>
 2: _napi_register_module_v1
    at <unknown source file>
 3: _napi_register_module_v1
    at <unknown source file>
 4: _napi_register_module_v1
    at <unknown source file>
 5: _napi_register_module_v1
    at <unknown source file>
 6: _wasmer_vm_imported_memory32_atomic_notify
    at <unknown source file>
 7: _napi_register_module_v1
    at <unknown source file>
 8: _napi_register_module_v1
    at <unknown source file>
 9: _napi_register_module_v1
    at <unknown source file>
10: _napi_register_module_v1
    at <unknown source file>
11: _napi_register_module_v1
    at <unknown source file>
12: _napi_register_module_v1
    at <unknown source file>
13: _wasmer_vm_imported_memory32_atomic_notify
    at <unknown source file>
14: _napi_register_module_v1
    at <unknown source file>
15: _napi_register_module_v1
    at <unknown source file>
16: _napi_register_module_v1
    at <unknown source file>
17: _napi_register_module_v1
    at <unknown source file>
18: __pthread_joiner_wake
    at <unknown source file>
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command was killed with SIGABRT (Aborted): rspack -c ./config.mjs
```

The imported file the that causes this error has the following contents:
```js
"use strict";

exports.__esModule = true;

exports.default = function (file, acceptedFiles) {
  if (file && acceptedFiles) {
    var acceptedFilesArray = Array.isArray(acceptedFiles) ? acceptedFiles : acceptedFiles.split(',');
    var fileName = file.name || '';
    var mimeType = (file.type || '').toLowerCase();
    var baseMimeType = mimeType.replace(/\/.*$/, '');
    return acceptedFilesArray.some(function (type) {
      var validType = type.trim().toLowerCase();

      if (validType.charAt(0) === '.') {
        return fileName.toLowerCase().endsWith(validType);
      } else if (validType.endsWith('/*')) {
        // This is something like a image/* mime type
        return baseMimeType === validType.replace(/\/.*$/, '');
      }

      return mimeType === validType;
    });
  }

  return true;
};
```

### Reproduce link

https://github.com/JohnDaly/rspack-concatenate-modules-panic-repro

### Reproduce Steps

1. Run `npm run build:rspack`
2. See crash output in terminal

**Comment by JohnDaly** [comment]
Seems related to https://github.com/web-infra-dev/rspack/issues/6871

**Comment by JohnDaly** [comment]
It looks like this will be fixed in a future version of `rspack`: https://github.com/web-infra-dev/rspack/issues/6871#issuecomment-2181984931

I updated `@rspack/core` to `0.7.5-canary-129960e-20240620181737` and the production build completes successfully

**Comment by chenjiahan** [comment]
Yes, this has been fixed and we will release `1.0.0-alpha.0` tomorrow
